configuration/pih/scripts/global/sampleCollection.js

```diff
@@ -0,0 +1,58 @@
+$(() => {
+	(async () => {
+	  // Sección: Arreglo de UI
+	  document.querySelector("#iframe > legend").innerText = "Solicitud de Análisis";
+
+	  // Sección: Llamadas a API - EMR
+	  const urlParams = new URLSearchParams(window.location.search);
+	  const encounterId = urlParams.get("encounterId"), patientId = urlParams.get("patientId");
+	  // Subsección: Encounter - Orders
+	  const encounter = await $.get(`/openmrs/ws/rest/v1/encounter/${encounterId}`);
+	  //examenes=orders.display;
+	  let fechaVisitaValue=new Date(Date.parse(encounter.encounterDatetime)).toLocaleString();
+	  let examenes=[];
+	  //console.log(encounter);
+	  $("#fecha").text(new Date(Date.parse(encounter.encounterDatetime)).toLocaleString());
+	  $("#local").text(encounter.location.display);
+	  $("#proyecto").text(encounter.location.display);
+	  $.each(encounter.orders, (i, orders) => {
+		//console.log(orders);
+		examenes.push(orders.display);
+		$("#examenes").append($(`<td>${orders.display}</td>`));
+	  });
+	  const paciente= await $.get(`/openmrs/ws/rest/v1/patient/${patientId}`);
```

Comment:
Please put spaces around assignment operators.

```suggestion
	  const paciente = await $.get(`/openmrs/ws/rest/v1/patient/${patientId}`);
```